src/test/java/uk/gov/justice/digital/delius/service/ConvictionServiceTest.java

```diff
@@ -292,7 +292,19 @@ public void convictionReturnedWhenSingleConvictionMatchedForOffenderIdButNotClos
 
             final var maybeConviction = convictionService.getSingleActiveConvictionIdByOffenderIdAndCloseToSentenceDate(99L, LocalDate.of(2020, 1, 21));
 
-            assertThat(maybeConviction.get()).isPresent();
+            assertThat(maybeConviction.get()).isEmpty();
```

Comment:
sadly the test was 100% wrong (the production code was 50% wrong)

Reply:
lol